test/inputs/actions.spec.js

```diff
@@ -0,0 +1,65 @@
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import nock from "nock";
+
+import * as ActionCreators from "../../src/inputs/actions";
+
+const middlewares = [ thunk ];
+const mockStore = configureStore(middlewares);
+
+describe("Action Creators", () => {
+    it("updateInputValue", (done) => {
+        const outriggerUrl = "http://localhost:3000";
+        const bundle = {
+            program: "input a: text; input b: dropdown -items [{value: 1, label: 'one'}, {value: 2, label: a}]; emit -limit 1"
+        };
+
+        const getState = {
+            outriggerUrl: outriggerUrl,
+            bundle,
+            inputs: [
+                {
+                    id: "a",
+                    value: null
+                },
+                {
+                    id: "b",
+                    value: null
+                }
+            ]
+        };
+
+        let mockResponseBody = [
+            {
+                id: "a",
+                value: "new value for a"
+            },
+            {
+                id: "b",
+                value: null,
+                options: {
+                    items: [{value: 1, label: "one"}, {value: 2, label: "new value for as"}]
+                }
+            }
+        ];
+
+        const action = {
+            type: "INPUT_DEFS_UPDATE",
+            payload: mockResponseBody
+        };
+        const expectedActions = [action];
```

Comment:
Changed, more a relic of my copying the example from redux-mock-store than intentionally doing it.